tools/test/h5repack/CMakeTests.cmake

```diff
@@ -314,7 +323,7 @@
             NAME H5REPACK-${testname}
             COMMAND ${CMAKE_COMMAND} -E echo "SKIP ${ARGN} ${PROJECT_BINARY_DIR}/testfiles/${testfile} ${PROJECT_BINARY_DIR}/testfiles/out-${testname}.${testfile}"
         )
-        set_property(TEST H5REPACK-${testname} PROPERTY DISABLED)
+        set_property(TEST H5REPACK-${testname} PROPERTY DISABLED true)
```

Comment:
Is this missing if ("H5REPACK-${testname}" MATCHES "${HDF5_DISABLE_TESTS_REGEX}")?  When I tested, not specifying H5REPACK nor H5IMPORT, there were 25 H5REPACK tests and 2 H5IMPORT tests that were disabled.